src/package.xml

```diff
@@ -203,6 +203,8 @@
         <members>Account.HEDA_Account_Compact_Layout</members>
         <members>Address__c.HEDA_Address_Compact_Layout</members>
         <members>Affiliation__c.HEDA_Affiliation_Compact_Layout</members>
+        <members>Behavior_Involvement__c.Behavior_Involvement_Compact_Layout</members>
```

Comment:
@xinnanlisf Should we fix the naming here to be consistent? "EDA_"?

Reply:
Since these are on legacy objects I'm gonna say we leave it. Pretty sure we decided that only net new page layouts would get the EDA prefix.

Reply:
Well, the Behavior one seems new. I'd consider giving that one a prefix. The others can't really be changed at this point, in case we were thinking moving from HEDA to EDA, since that would create brand new ones.

Reply:
Oh yea, I was looking at the HEDA stuff and made an assumption...yes we should add the EDA prefix to the Behavior one...my coffee isn't working apparently.

Reply:
No worries. I had the same initial thought when I got the email notification, since you can't see the colors there. Only when I came to Github I realized. :-)

Reply:
Sorry, I'll be more clear next time!

cc @xinnanlisf in case you didn't see this :)

Reply:
should be good now!